docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
 
 # IPFS and Filecoin
 
-Filecoin and IPFS are complementary protocols for storing and sharing data in the decentralized web. While users aren't required to use Filecoin and IPFS together, the two combined solve the current web infrastructure's significant failings. This page aims to explain the relationship between the two protocols and assist users in deciding which option, or combination of options, is best suited for their use-case.
+Filecoin and IPFS are complementary systems for storing and sharing data in the distributed web. Both are free, open-source and share a number of building blocks, including data representation formats and certain protocols for network communication. However, the Filecoin and the IPFS networks are otherwise fully independent. Software that interacts with IPFS does not require Filecoin, and vice-versa, although some solutions may make use of both.
 
-## Incentivizing data reliability
+This page aims to explain the relationship between the IPFS and Filecoin and assist users in deciding which option, or combination of options, is best suited for their use-case.
 
-[IPFS](https://ipfs.io) is a free and open-source network protocol that allows users to store and transfer verifiable data with each other. IPFS users persist data on the network by [pinning](https://docs-beta.ipfs.io/concepts/persistence/#pinning-in-context) it on their own hardware, to a third-party pinning service, or through groups of individual IPFS users sharing resources to ensure the content remains live.
+## Data storage incentives
 
-The lack of a built-in incentive mechanism is the challenge Filecoin hopes to solve by allowing users to incentivize long-term distributed storage at competitive prices through a marketplace of storage nodes while maintaining the efficiency and resiliency provided by the IPFS network.
+[IPFS](https://ipfs.io) allows users to store and transfer verifiable, content-addressed data in a peer-to-peer network. IPFS users persist the data they want **on their own IPFS nodes**. This is referred as [pinning](https://docs.ipfs.io/concepts/persistence). Sometimes the data may be pinned using a third-party pinning service, or through groups of individual IPFS users. The data exists in the network as long as one user is storing it and able to provide it to others when they request it.
```

Comment:
There are subtleties here. `can provide it to others when requested` is ambiguous as it does not specify who requests it. It would seem as if the node storing it needs to request it.

I also don't fully understand why we remove all bold highlighting. There has to be a way to make certain key pieces of information stand out in a paragraph, it is not something we abuse.